app/va/vetext/client.py

```diff
@@ -29,72 +31,71 @@ def init_app(
+            formatted_payload['icn'] = payload.icn
+        return formatted_payload
+
     def send_push_notification(
         self,
-        mobile_app: str,
-        template_id: str,
-        identifier: str,
-        is_broadcast: bool = False,
-        personalization: Dict = None,
+        payload: dict[str, str],
     ) -> None:
-        """
-        If "is_broadcast" is True, "identifier" is the "topicSid".  Otherwise, it is the "icn".
-        """
-
-        formatted_personalization = None
-        if personalization:
-            formatted_personalization = {}
-            for key, value in personalization.items():
-                formatted_personalization[f'%{key.upper()}%'] = value
+        """Send the notification to VEText and handle any errors.
 
-        payload = {
-            'appSid': mobile_app,
-            ('topicSid' if is_broadcast else 'icn'): identifier,
-            'templateSid': template_id,
-            'personalization': formatted_personalization,
-        }
+        Args:
+            payload (dict[str, str]): The data to send to VEText
+        """
         self.logger.debug('VEText Payload information: %s', payload)
-
+        start_time = monotonic()
         try:
-            start_time = monotonic()
             response = requests.post(
                 f'{self.base_url}/mobile/push/send', auth=self.auth, json=payload, timeout=self.TIMEOUT
             )
             self.logger.info('VEText response: %s', response.json() if response.ok else response.status_code)
             response.raise_for_status()
         except requests.exceptions.ReadTimeout:
-            # Discussion with VEText: read timeouts are still processed, so are marking this as good
-            self.logger.exception('ReadTimeout raised sending push notification - Still processed, returning 201')
-            # Logging as error.read_timeout so we can easily track it
+            # Discussion with VEText: read timeouts are still processed, so no need to retry
+            self.logger.info('ReadTimeout raised sending push notification - notification still processed')
             self.statsd.incr(f'{self.STATSD_KEY}.error.read_timeout')
         except requests.exceptions.ConnectTimeout as e:
-            self.logger.exception('ConnectTimeout raised sending push notification - Retrying')
-            # Logging as error.read_timeout so we can easily track it
+            self.logger.warning('ConnectTimeout raised sending push notification - Retrying')
             self.statsd.incr(f'{self.STATSD_KEY}.error.connection_timeout')
-            raise VETextRetryableException from e
+            raise RetryableException from e
         except requests.HTTPError as e:
-            self.logger.exception('HTTPError raised sending push notification')
             self.statsd.incr(f'{self.STATSD_KEY}.error.{e.response.status_code}')
             if e.response.status_code in [429, 500, 502, 503, 504]:
+                # TODO - #2172 DEV Failing Here
```

Comment:
Failing here - Appears there's something wrong with how the error is raised `raise RetryableException from e`